turbo.json

```diff
@@ -3,7 +3,7 @@
   "pipeline": {
     "build": {
       "outputs": ["build/**", ".next/**", "!.next/cache/**"],
```

Comment:
Thought don't generated files need to be specified in the outputs list so they are zipped into the cache and unzipped to be replayed?

Reply:
I've included the output files in the [`generate`](https://github.com/osmosis-labs/osmosis-frontend/blob/493c199aecd3b139298c5b06aa82fddefe20f592/turbo.json#L15-L23) key. As it executes before the build process, it should retrieve the files stored in the cache

Reply:
Ah missed that. Nice 👍